### PR TITLE
Bluespace Admin Lockers??!?!?!?!?!

### LINKED ISSUE #9314: Can expand bluespace locker into centcom z-level via bluespace shelter capsules
## Reproduction:
Simply drop bluespace shelter capsules on the locker's edges, RCD the walls and repeat until one could simply walk into Centcom z-level. Admins responded accordingly:
image
(Round ID: 30767)

### PR DESCRIPTION
**Pull request by JamieD1**
Bluespace Admin Lockers??!?!?!?!?!

### Intent of your Pull Request

Fixes: #9314 

#### Changelog

:cl:  
bugfix: Bluespace Admin Lockers??!?!?!?!?!
/:cl:

### CODE REVIEW

File: code/modules/mining/shelters.dm [hunk 1]
```diff
@@ -17,9 +17,8 @@
 	var/affected = get_affected_turfs(deploy_location, centered=TRUE)
 	for(var/turf/T in affected)
 		var/area/A = get_area(T)
-		if(is_type_in_typecache(A, banned_areas))
+		if(is_type_in_typecache(A, banned_areas) || T.z == 1)
```
Comment (SomeguyManperson) [inline code/modules/mining/shelters.dm]: add /area/bluespace_locker to the banned_areas list ez

Comment (JamieD1) [reply code/modules/mining/shelters.dm]: No because that would mean I need to add every centcom area

**Comment by UselessTheremin** [comment]
i was actually there that round! we were free miners, and the captain was the one doing it. i am against removing this so admins can have the joy of smiting people who come to centcomm.

File: code/modules/mining/shelters.dm (hunk 1 above)
Comment (boodaliboo) [reply code/modules/mining/shelters.dm]: would that stop abductor teleports?

Comment (Amelia0010) [reply code/modules/mining/shelters.dm]: change https://github.com/yogstation13/Yogstation/blob/ab6808b76f907488f028f55d733ee6eb563c7868/code/modules/mining/equipment/survival_pod.dm#L46 to get_turf_global instead so it just works outside of the bluespace locker

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**